requirements/pymeasure.yml

```
@@ -16,7 +16,7 @@ dependencies:
   - pytest-qt=3.3.0
   - pytest-runner=5.2
   - pytest=6.2.5
-  - flake8=4.0.1
+  - flake8=5.0.4
```

```suggestion
  - flake8=5.0.4 # can be bumped to v6 as soon as we drop python 3.7
```
To document the constraint we have, and leave a breadcrumb for things to do when updating the minimum python version.

---

Let us think, how we put these hints into the code, that we might find them more easily. Silimar to #672
Elsewhere I used "py>3.7" as a code word.

---

TBH, I just "search in all files" for `3.7` and `37`. The first one typically catches all but one or two, false positives should be few.

---

But still the question remains, whether the last version or the next version is mentioned ("bump version, when we drop 37" or "minimum version 38")

---

Good idea. Since it concerns dropping support for a python version, it makes most sense to me to mention the version that is dropped (so 3.7 in this case).

---

I just saw, that we already talked about that in #673 
Your suggestion is reasonable, I'll add it to #672 

From my side, this conversation can be resolved.